### DISCUSSION TITLE
Should the project be directed towards minimalism or towards productivity?

### DISCUSSION THREAD

**Discussion by AshiVered** [AshiVered/SkyOS#7] [Polls]
After a long time, the project was [unfrozen](https://github.com/AshiVered/SkyOS/issues/6).

And I'm interested in making quite a few changes.

The project was originally intended to be minimalist - just remove the bloatware and make the device faster.

But... now I thought of taking the project in a productive direction:
For example, to add OsmAnd maps, PDF reader, etc.

what do you think?
If you are in favor of the productive direction, I would also appreciate ideas for applications that should be integrated.

**Comment by LolloDev5123** [comment]
I think a PDF Reader is too much bloat, however, a GPS/Nav App like OSM would be appropriate for the N800T
